docs/core/versions/index.md

```diff
@@ -27,6 +27,16 @@ With .NET Core 2.0, downloads show a single version number in their file name. T
 
 The use of a single version number makes it easier for users to know what version of the SDK to install on their dev machines, and what the corresponding version of the shared framework should be when time comes to provision a production environment. When downloading an SDK or runtime, the version number you see is going to be the same.
```

Comment:
just a heads up. I think this is the article I'm rewriting. 

"version manager" feels like quite a stretch for our experience. Since I want to build that, I'd rather change hte first bullet here

Reply:
Yeah I think this whole document needs to be reviewed / rewritten.  It sounds like Kathleen is doing so.

To be clear, there is **not** a single version number for both the runtime and the SDK :)